geometry_msgs/PoseStamped goalPose
uint64 minExecutionTick